---

---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE)
```

# UMAMI RECEPTOR PROJECT
## INTRODUCTION  
During hominin evolution, it has been suggested that the introduction of meat into the hominin diet provided a new, calorically rich food source that resulted in a higher daily caloric intake. Human brains have intense energetic demands (~20-25% of resting metabolic rate) compared to the average primate (~9%) and the average mammal (~3-5%) (Leonard, 2007) and the caloric surplus produced from a dietary transition towards more regular meat eating may have increased the calories available to allocate towards expensive organs such as the brain. Although humans present an extreme example of brain size, it should be noted that, generally, primates are more encephalized than the average mammal. Indeed, across primates, diet and brain size appear to be tightly linked, although researchers debate the dietary drivers of this increased brain size. Recent work has suggested that frugivorous primates have larger brains (DeCasien, 2017), other work has suggested that lipid dense foods such as nuts may be drivers (Izar, 2022). However, the work from DeCasien et al. explored primates predominately through a binary of percent folgivorous diet versus percent frugivorous diet, which is not representative of the heterogeneity of the primate diet and ignores primate participation in meat eating behaviors with humans being the most prolific meat eaters. Some primates are known to eat meat semi-frequently, such as the chimpanzee, baboon and Capuchin, although many primates have been observed with occasional, opportunistic meat eating of small vertebrates such as amphibians, birds and bats (Watts, 2020). While its possible that fruit intake may drive the general encephalized pattern across primates, as DeCasien et al. suggest, the human brain is much more energetically demanding than other primate brains, and its possible that another dietary transition needed to occur to facilitate this brain expansion in humans. Humans are outliers both in terms of frequency of meat eating, but also in the processing of animal remains, particularly in the processing of lipid-rich bone marrow via tool use. Indeed, early humans participated in a unique dietary niche of meat exploitation and it appears this dietary incorporation of meat preceded hominin brain expansion (Ferraro, 2013). The degree of meat exploitation and the additional processing of bone marrow appear to be unique features of the hominin dietary niche and, I hypothesis, that this dietary transition to meat proteins/lipids was a primary diver in supporting the energetics which underpin hominin brain evolution as opposed to other food sources. *[More info related to increased caloric intake vs changing macronutrient profiles to come in next homework]* **To evaluate the importance of meat relative to other food sources in hominin evolution, this project compares genes related to the taste perception and digestion of protein across primate lineages in order to evaluate selection on these genes. Specifically, this project for this class explores two genes, TAS1R1 and TAS1R3, which form the umami taste receptor heterodimer complex which is associated with human's taste perception of meat.** 

## PROJECT LIMITATIONS / FUTURE ACTIONS
This project serves as an introduction into a larger project which I aim to accomplish over this coming summer. This larger project will explore not only the TAS1R1 & TAS1R3 genes, but will also include other genes known to be linked to protein digestion. Given the work suggesting the importance of fruit in brain expansion across primates, it may be worthwhile to explore genes related to both protein and fruit digestion. It could be interesting to see if more meat-related genes experience selection in the hominin lineage as opposed to fruit-related genes, and also evaluate if the inverse trend appears in other primates. The particulars of this project are still under revision, however, its important to stress that exclusive exploration of only the TAS1R1 and TAS1R3 genes is not sufficient to support or reject the importance of meat in the hominin lineage. Both TAS1R1 and TAS1R3 have a complicated history across primates, as described in more detail below, where evidence suggest that these genes may serve different functional purposes in other primates, meaning they may not be used to transduce umami flavor and thus may be selected upon for other dietary specializations. However, if a host of genes related to protein digestion as well as the TAS1R1 and TAS1R3 genes experience selection in the hominin lineage, this could provide strong support for the hominin meat eating hypothesis. Therefore, while this current project only considers TAS1R1 and TAS1R3 genes, this acts as a trial run for a project with a larger scope and, hopefully, more explanatory power. 

## UMAMI RECEPTOR GENES
While there have been several other genes implicated in the perception of umami flavor, much research has focused on the TAS1R1 and TAS1R3 genes, which form a T1R1/T1R3 heterodimer responsible for the transduction of umami flavor. Umami flavor is described as the "savory" component to foods, and is associated with the flavors of cooked meats, specific cheeses (Diepeveen, 2022) and fermented foods ( Diepeveen, 2022; Zhang, 2019). This receptor is a G protein coupled receptor (GPCR) which, in humans, is activated by binding of monosodium glutamate (MSG), free L-amino acids and peptides (Zhang, 2019). However, research indicates the responsivity of these receptors to varying substances differs among primates. For example, Todo et al. cloned TAS1R1 receptors of numerous primates across the primate phylogeny and exposed these receptors to L-glutamate (L-glu) and 5'ribonucleotides. They found folivorous primate receptors were responsive to L-glu, which is a prominent component of leaves, and insectivorous primate receptors were responsive to 5' ribonucleotides, which is a prominent component of insects. The receptors of primates with more diverse diets, termed dietary generalists, were responsive to both L-glu and 5' ribonucleotides (Toda, 2021). This research indicates that the TAS1R1 and, possibly, the TAS1R3 receptor serve purposes beyond meat perception in other primates and, thus, this  complicates the examination of these genes in the hominin lineage compared to other primate lineages. While this complicated history must be considered in analyses, exploration of the TAS1R1 and TAS1R3 genes in the hominin lineage still has the potential to reveal interesting insights. For example, research has identified key residues important for the binding of MSG, inosine monophosphate (IMP), sodium succinate (WSA) and "beefy meaty peptide" (BMP). For TAS1R1, there are six key residues important for BMP binding which include N69, Y220, S276, R277, A302, and S385. For TAS1R3, there is one key residue which is R151 (Diepeveen, 2022). Selection on sites responsible for binding of meat-related ligands in the hominin lineage could be informative of meat's relative importance. *[More details will need to be explored here]*

**HYPOTHESES**

*[Plans to integrate exploratory analysis of gene tree and free-omega branch model to access level of variation in dN/dS across lineages--will be addressed in next homework]*

In humans, while umami flavor can be associated with other foods, its most commonly associated with cooked meat. Given the suggested importance of meat in the hominin lineage, it is possible these genes have experienced selection: 

**H1: TAS1R1 and/or TAS1R3 will show signatures of positive selection in the human lineage, as determined by comparison of the human foreground branch model results and the null model results via a likelihood ratio test.**

As several other primates also partake in meat eating behaviors, these primates may also show positive selection on either receptor gene:

**H2: Primates known to participate in meat eating behaviors, such as the chimpanzee, will show selection on TAS1R1 and/or TAS1R3, as determined by comparison of the foreground branch model results and the null model results via a likelihood ratio test.**


In the broader primatological context, these receptors appear to serve several dietary functions. For example, the TAS1R1 receptor of dietary specialists, such as fully folivorous or insectivorous primates, demonstrate highly specific and exclusive responses to the molecules that comprise their specialized diets. This, perhaps, suggests heightened importance of these receptors to dietary specialists: 

**H3: TAS1R1 and/or TAS1R3 will show signatures of positive selection in predominately insectivorous primates such as the tarsier, as determined by comparison of the tarsier foreground branch model results and the null model results via a likelihood ratio test.**

**H4: TAS1R1 and/or TAS1R3 will show signatures of positive selection in predominately folivorous primates such as the gorilla, as determined by comparison of the tarsier foreground branch model results and the null model results via a likelihood ratio test.** 

As specific amino acid residues were identified as important in facilitating a strong response to either L-glu or 5'ribonucleotides in Toda et al., I anticipate that the evaluation of these sites will show selection:  

**H5: Both predominately insectivorous primates and predominately folivorous primates will demonstrate selection at the sites 170 and 302 of the TAS1R1 gene, as determined by Bayes Empirical Bayes method.** 


While all above hypotheses have suggested positive selection, its possible that primates with very general diets or perhaps predominately frugivorous diets may not show selection:

**H6: TAS1R1 and/or TAS1R3 will not show signatures of selection in dietary generalists who do not exhibit meat eating behaviors, as determined by comparison of the foreground branch model results and the null model results via a likelihood ratio test.**

# METHODS

## TAS1R1 AND TAS1R3 GENE HISTORY
Both TAS1R1 and TAS1R3 were examined on ensembl.org. For both genes, all primate species had 1:1 orthologue status, meaning only one copy was found in each of species. To further examine the possibility of gene duplication events or any other complexity of gene history confounding analyses of these genes, I separately ran both the TAS1R1 and TAS1R3 human amino acid transcripts through BlastP to check the sequences that returned with high similarity. For both genes, all returning sequences were labeled as the gene of interest. There did not appear to be any other sequences returning. However, some species did have multiple isoforms of the gene. I individually checked each primate from Ensembl.org and some species did have 2-4 known splice variants for either gene, so these results were not surprising.


# COMPLETED METHODS
DNA and amino acid sequence data for the TAS1R1 gene (ENSG00000173662) and the TAS1R3 gene (ENSG00000169962) were gathered from ensembl.org on February 8th, 2023. Ensembl identified 22 species of primates with orthologous sequences for TAS1R1 and 19 species of primates with orthologous sequences for TAS1R3. Both the amino acid sequence and the DNA sequence were downloaded and aligned using muscle. Following this alignment, all gaps in the DNA were checked and manually adjusted based on the amino acid alignment to ensure proper reading frame of codons. The TAS1R1 alignment was noted to be of high quality with only Neumascus leugenys (Gibbon) needing adjustment. The TAS1R3 alignment showed more mutations between species. Particularly, Roxellana bieti (black snub nose monkey) showed numerous insertions and deletions throughout the alignment. Although the subsequent steps were performed with the black snub nose monkey in the TAS1R3 alignment, future re-analyses may remove this species to see if it impacts the results, as this primate appears to be an extreme outlier. Aside from the black snub nose monkey, the TAS1R3 alignment was also of good quality. Following DNA alignment, the alignments for TAS1R1 and TAS1R3 were input in PAUP and both underwent a Maximum Likelihood bootstrap analysis (500 replicates). The resultant trees showed that all primates grouped together based on relatedness for both genes (i.e. New World Monkeys formed a clade, Old World Monkeys formed a clade, etc). The majority of bootstrap values were greater than 95 with the exception of four values for TAS1R1 (52.2, 54, 76.8, and 86.6) and four values for TAS1R3 (58.6, 60.4, 63.8 and 81.6). Additionally, the TAS1R1 and TAS1R3 alignment files were combined then input together into PAUP, where they underwent a Maximum Likelihood bootstrap (500 replicates) as described above. The prosimian TAS1R1 gene was the outgroup. Following this, the TAS1R1 and TAS1R3 genes split and, following this split, both genes grouped via relatedness. These trees can be viewed below. More detailed explanations of these steps were documented in README.md files and can be provided upon request.


# FUTURE METHODS
The tree and alignments will be analyzed in PAML to evaluate possibility of selection. I have spent the last two weeks using example data sets to test proper usage of PAML (general protocols evaluated from Bielawski 2013, Bielawski 2016, Yang 1997, and Yang 1998). I have completed test runs of my alignment and trees in PAML. Although results were generated from these runs, there was an error message regarding an abrupt end of tree file which I am currently trouble shooting (I feel confident that making adjustments to tree file will resolve the error). Once the program runs without the error message, I anticipate uusing an interative R script which will run on Argon. I wil luse both PAML's branch model and branch site model. PAML's branch model will be used to evaluate possible selection on primate lineages. Each primate species will be treated as the foreground branch in independent runs. Additionally, human and all non-human primate lineages will undergo branch-site model tests. I anticipate results in line with previous hypotheses. 
 

# ANTICIPATED IMAGE 
Following these PAML analyses, I will use R to construct a primate phylogeny which will document type of selection (negative, neutral or positive) on each primate branch (see Image 2 for the image of inspiration). I anticipate importing a phylogenetic tree into R, then I will use packages such as ggtree in R to modify the tree. As in my inspiration image, I anticipate using color coordinated node labels to indicate negative, neutral or positive selection on that branch. This can be done in R. As far as the vertical group labels, I will attempt to find functions in R to acheive this look, although at this moment I am unaware of such functions or aesthetics. However, if I am unable to produce this in R, I will export the image with its colored nodes then make the subsequent additions (the side label groupings) in powerpoint.


# REFERENCES
Bielawski, Joseph P. "Detecting the signatures of adaptive evolution in protein-coding genes." Current protocols in molecular biology. January 15, 2013, 101. DOI: https://doi.org/10.1002/0471142727.mb1901s101.

Bielawski, Joseph P., Jennifer L. Baker, and Joseph Mingrone. “Inference of episodic changes in natural selection acting on protein coding sequences via CODEML.” Current protocols in bioinformatics. June 20, 2016, 54. DOI: https://doi.org/10.1002/cpbi.2.

DeCasien, A. R., Williams, S. A., & Higham, J. P. "Primate brain size is predicted by diet but not sociality." Nature ecology & evolution, 2017, 1(5). DOI: https://doi.org/10.1038/s41559-017-0112. 

Diepeveen, J., Moerdijk‐Poortvliet, T. C., & van der Leij, F. R. "Molecular insights into human taste perception and umami tastants: A review." Journal of Food Science, 2022, 87(4). DOI: https://doi.org/10.1111/1750-3841.16101.

Ferraro, Joseph V., Thomas W. Plummer, Briana L. Pobiner, James S. Oliver, Laura C. Bishop, David R. Braun, Peter W. Ditchfield et al. “Earliest archaeological evidence of persistent hominin carnivory.” PloS one 8, April 25, 2013, 4. DOI:10.1371/journal.pone.0062174.

Izar, P., Peternelli-dos-Santos, L., Rothman, J. M., Raubenheimer, D., Presotto, A., Gort, G., ... & Fragaszy, D. M. (2022). "Stone tools improve diet quality in wild monkeys." Current Biology, 2022, 32(18). DOI: https://doi.org/10.1016/j.cub.2022.07.056.

Leonard, William. R., Josh J. Snodgrass, and Marcia L. Robertson. “Effects of brain evolution on human nutrition and metabolism.” Annu. Rev. Nutr, August 21, 2007, 27. DOI: 10.1146/an- nurev.nutr.27.061406.093659.

Toda, Yasuka, et al. "Evolution of the primate gluatamate taste sensor from a nucleotide sensor." Current Biology. October 25, 2021, 31. DOI:https://doi.org/10.1016/j.cub.2021.08.002. 

Watts, David P. "Eating eating by nonhuman primates: A review and synthesis." Journal of Human Evolution. October 24, 2020, 149. DOI:Https://doi.org/10.1016/j.jhevol.2020.102882. 

Wu, Yonghua, Haifeng Wang, Haitao Wang, and Elisabeth A. Hadly. "Rethinking the origin of primates by reconstructing their diel activity patterns using genetics and morphology." Scientific Reports, September 19 2017, 7(1). DOI: https://doi.org/10.1038/s41598-017-12090-3. 

Zhang, Jianan, Dongxiao Sun-Waterhouse, Guowan Su, & Mouming Zhao. "New insight into umami receptor, umami/umami-enhancing peptides and their derivatives: A review." Trends in Food Science & Technology, Jun 1, 2019. DOI: https://doi.org/10.1016/j.tifs.2019.04.008.

Zhang, Jianzhi. “Parallel Adaptive Origins of Digestive RNases in Asian and African Leaf Monkeys.” Nature Genetics, July 2006, 38(7). DOI: https://doi.org/10.1038/ng1812.

Yang, Ziheng. "PAML: a program package for phylogenetic analysis by Maximum likelihood." Computer applications in the biosciences, October 1, 1997, 13(5). DOI: 10.1093/bioinformatics/13.5.555. 

Yang, Ziheng. “Likelihood Ratio Tests for Detecting Positive Selection and Application to Primate Lysozyme Evolution.” Molecular Biology and Evolution, May 1998, 15(5). https://doi.org/10.1093/oxfordjournals.molbev.a025957.


# APPENDIX 
![Anticipated results based on hypotheses](hypothesis_expectations.jpg)

![As this research has not been completed, I have shown an example of a similar phylogeny from Wu et al. which was a reconstruction of primate diel pattern using opsin genes. Using the results from my PAML analyses, I will create a similar phylogenetic tree where I mark branches with their respective selection symbol.](Homework0_PrimatePhylogenyImage.jpg)

![The maximum likelihood tree for the TAS1R1 gene](Human_TAS1R1_orthologues_DNA_aligned_tree.jpg)

![The maximum likelihood tree for the TAS1R3 gene](Human_TAS1R3_orthologues_DNA_aligned_tree.jpg)

![The maximum likelihood tree for both the TAS1R1 gene and the TAS1R3 gene when they were combined into a single file.](Human_TAS1R13_orthologues_DNA_aligned_combined_tree.jpg)
